packages/ui5-application-writer/src/data/defaults.ts

```diff
@@ -225,3 +226,56 @@ function getLocalVersion({
     }
     return result;
 }
+
+function runCommand(cmd: string, args: string[] = [], opts: object = {}): Promise<string | void> {
+    return new Promise((resolve, reject) => {
+        const stack: string[] = [];
+        const command = `\`${cmd} ${args.join(' ')}\``;
+
+        const spawnedCmd = spawn(cmd, args, opts);
+        let response: string;
+
+        spawnedCmd.stdout.on('data', (data) => {
+            stack.push(data.toString());
+            response = data.toString();
+        });
+        spawnedCmd.stderr.on('data', (data) => {
+            stack.push(data.toString());
+        });
+        spawnedCmd.on('close', (errorCode) => {
+            if (errorCode !== 0) {
+                return reject(` Error ${errorCode} running ${command}\n${stack.join(', ')}`);
+            }
+            resolve(response);
+        });
+    });
+}
+
+/**
+ * Retrieve the tag version of the @sap/ux-specification based on the given version.
+ *
+ * @param ui5Version UI5 version used in the project
+ * @returns version tag
+ */
+export async function getSpecTagVersion(ui5Version: string | undefined): Promise<string> {
```

Comment:
@tobiasqueck, for each UI5 version there is a `dist-tag` for module `@sap/ux-specification`, see: https://www.npmjs.com/package/@sap/ux-specification -> Versions tab

Name pattern is `UI5-{major}.{minor}`, e.g. `UI5-1.103`. Each `dist-tag` points to the respective `@sap/ux-specification` version.

This means, there is no need to call `npm view`. As long as you know the UI5 version, you can build the `dist-tag` and write it to `package.json`, e.g.:
```
"devDependencies": {
    "@sap/ux-specification": "UI5-1.103"
    ...
```

if no UI5 version is set, you can use `latest`, that is then aligned with `"minUI5Version" : ""`.

Reply:
@Klaus-Keller thanks a lot. I ported that code from our Fiori generator assuming it is the best way doing it.

Reply:
Fixed with https://github.com/SAP/open-ux-tools/pull/629/commits/fcb0770d22aee090410919d20ee8e270b497a7bc.
@Klaus-Keller @devinea please close the conversation if this is acceptable for you.

Reply:
@tobiasqueck  thanks for https://github.com/SAP/open-ux-tools/commit/fcb0770d22aee090410919d20ee8e270b497a7bc